src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const costDayCar = 40;
+  const shortTerm = 3;
+  const longTerm = 7;
+  const lessWeeklyDiscount = 20;
+  const weeklyDiscount = 50;
+
+  if (days < shortTerm) {
+    return costDayCar * days;
+  } else if (days < longTerm) {
+    return costDayCar * days - lessWeeklyDiscount;
+  }
+
+  return costDayCar * days - weeklyDiscount;
```

Comment:
1. move costDayCar * days to a variable
2. when you use `early return` inside `if` you can avoid using `else`
3. first condition should be for longTerm and then for shortTerm